### DISCUSSION TITLE
Execute Stored Procedure

### DISCUSSION THREAD

**Discussion by tarun-kreativeapps** [directus/directus#11291] [Archive]
Hi, I am new with DirectUs and trying to integrate it with my small project application. I have some complext queries to make. Can we execute stored procedure using directus api?

Any help will be great!

**Comment by rijkvanzanten** [comment]
Heya! Directus doesn't currently have any stored procedure capabilities included out of the box. The alternative is to [create a custom endpoint](https://docs.directus.io/extensions/endpoints/) that runs the SQL directly 👍🏻
